src/script/components/InputBar/InputBar.tsx

```diff
@@ -90,6 +90,7 @@ import {UserState} from '../../user/UserState';
 const CONFIG = {
   ...Config.getConfig(),
   PING_TIMEOUT: TIME_IN_MILLIS.SECOND * 2,
+  IS_TYPING_TIMEOUT: TIME_IN_MILLIS.SECOND * 10,
```

Comment:
did product already agree on 10s? I feel like it's too long 😅

Reply:
me too but the product team has not agreed yet, how about making it 5 for now?